laddertools/ladder.py

```diff
@@ -151,7 +186,10 @@ def _main(args):
     # Re-use the cached OpenRA account information to prevent stressing too
     # much the service
     request_accounts = c.execute('SELECT * FROM accounts')
-    accounts_db = {fp: (pid, pname, avatar_url) for fp, pid, pname, avatar_url in request_accounts.fetchall()}
+    accounts_db = {
+        fp: (pid, pname, avatar_url)
+        for fp, pid, pname, avatar_url in request_accounts.fetchall()
+    }
```

Comment:
I don't mind this change but I believe it's unrelated to this commit

Reply:
Ah yes, it's a rebase to the master branch. The multiple lines is auto-formatting, but I can keep it also on one line. I just did it in case of any merge conflicts.